ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
I think we should address it and add a test. Rather see that work done now then have to remember to do it in the rewrite. /2cents